modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/Workflow.macro

```diff
@@ -875,20 +875,11 @@ definition {
 			panel = "Applications",
 			portlet = "Process Builder");
 
-		Navigator.gotoNavItem(navItem = "Schemes");
+		Navigator.gotoNavItem(navItem = "Configuration");
```

Comment:
I'm not too familiar with workflow so I wanted to ask a couple of questions.
"Schemes" to "Configuration". Will that cause problems for other workflow tests?